FLEDGE.md

```diff
@@ -540,8 +562,9 @@ generateBid(interestGroup, auctionSignals, perBuyerSignals,
   return {'ad': adObject,
           'adCost': optionalAdCost,
           'bid': bidValue,
-          'render': renderURL,
-          'adComponents': [adComponent1, adComponent2, ...],
+          'render': {url: renderURL, size: {width: renderWidth, height: renderHeight}},
```

Comment:
This looks inaccurate?  ConvertDomStringOrAdRender looks to expect {url:..., width:..., height:...} - that is, a single dictionary, not a nested size dictionary.  Same for adComponents below.

Reply:
Thanks for catching this! Looks like the implementation somehow desynced even though it was supposed to be based on this description. I'll just update the explainer since the formats are basically interchangeable.